apis/metal3.io/v1alpha1/baremetalhost_validation_test.go

```diff
@@ -437,6 +438,43 @@ func TestValidateCreate(t *testing.T) {
 			oldBMH:    nil,
 			wantedErr: "BMO validation: failed to parse BMC address information: BMC address hostname/IP : [fe80::fc33:62ff:fe33:8xff] is invalid",
 		},
+		{
+			name: "Invalidate image URL",
+			newBMH: &BareMetalHost{
+				TypeMeta:   tm,
+				ObjectMeta: om,
+				Spec: BareMetalHostSpec{
+					BMC: BMCDetails{
+						Address:         "idrac://127.0.0.1",
+						CredentialsName: "test1",
+					},
+					Image: &Image{
+						URL: "test1",
+					},
+				},
+			},
+			oldBMH:    nil,
+			wantedErr: "Image URL test1 is an invalid URL",
+		},
+		{
+			name: "Invalidate image URL",
```

Comment:
Do we need to update this as per `wantedErr` description?

Reply:
updated, thanks.